src/dmd/cparse.d

```diff
@@ -2644,6 +2681,15 @@ final class CParser(AST) : Parser!AST
         const loc = token.loc;
         nextToken();
 
+        /* GNU Extensions
+         * enum-specifier:
```

Comment:
Link to (the closest thing to a) specification from the horse's (gnu's?) mouth?

Reply:
The function cparseGnuAttributes has the link, I don't think it is worth repasting it everywhere.

For your review this is it.

https://gcc.gnu.org/onlinedocs/gcc/Attribute-Syntax.html

As you will see, it uses hand wavy words rather than clear, concise grammar to describe how it is parsed.

Reply:
@ibuclaw Must've missed that my bad, thanks for the good work